x/evm/keeper/grpc_query.go

```diff
@@ -0,0 +1,27 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/axelarnetwork/axelar-core/x/evm/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+var _ types.QueryServiceServer = baseKeeper{}
+
+func (k baseKeeper) BurnerInfo(c context.Context, req *types.BurnerInfoRequest) (*types.BurnerInfoResponse, error) {
+	ctx := sdk.UnwrapSDKContext(c)
+
+	if !k.HasChain(ctx, req.Chain) {
+		return nil, sdkerrors.Wrapf(types.ErrBurnerInfoNotFound, "unkown chain '%s'", req.Chain)
```

Comment:
Wouldn't this be `ErrChainNotFound`?

Reply:
Makes sense, but if we are gonna introduce such error code now, we should probably update the entire module with that error code, which kinda is outside the scope of this PR

Reply:
The chain parameter was removed from this query, so there is no longer a need to create such error code for the sake of this query. It still makes sense to create such error code and use it across the EVM module, but that falls outside the scope of this PR.